.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/Financial-Times/http-handlers-go
+    docker:
+      - image: golang:1.8.3
```

Comment:
dep requires 1.9+ version of go if I recall correctly

Reply:
But we don't build dep, it's just installed as a binary.

Reply:
👍